discovery-provider/alembic/trigger_sql/handle_follow.sql

```diff
@@ -34,12 +34,34 @@ begin
     on conflict do nothing;
   end if;
 
+  begin
+    -- create a notification for the followee
+    if new.is_delete is false then
+      insert into notification
+      (blocknumber, user_ids, timestamp, type, specifier, metadata)
+      values
+      (
+        new.blocknumber,
+        ARRAY [new.followee_user_id],
+        new.created_at,
+        'follow',
+        'follow:' || new.followee_user_id,
+        ('{ "followee_user_id": ' || new.followee_user_id || ',  "follower_user_id": ' || new.follower_user_id ||  '}')::json
+      );
+    end if;
+	exception
+		when others then
+		raise notice 'Error creating follower notification';
+	end;
```

Comment:
similar to above:
* `json_build_object`
* `on conflict do nothing` instead of exception handling